package.json

```diff
@@ -12,8 +12,8 @@
     "@mdx-js/react": "^1.6.22",
     "@ukic/docs": "^2.1.0-beta.11",
     "@ukic/fonts": "^2.1.0-beta.11",
-    "@ukic/react": "^2.1.0-beta.11",
-    "@ukic/web-components": "^2.1.0-beta.11",
+    "@ukic/react": "file:../../ic-ui-kit/packages/react/ukic-react-2.1.0-beta.8.tgz",
```

Comment:
```suggestion
    "@ukic/react": "^2.1.0-beta.11",
```
I think these just need changing back to how they were, but other than that, looks good 😊